### PR TITLE
added state attributes for WandB Logger

### PR DESCRIPTION
**Pull request by Ishan-Kumar2**
Addresses #2100

Description:
Adds feature to log state attributes in WandB
Check list:

- [X] New tests are added (if a new feature is added)
- [X] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

File: ignite/contrib/handlers/wandb_logger.py [hunk 1]
```diff
@@ -226,6 +226,22 @@ def global_step_transform(*args, **kwargs):
                 global_step_transform=global_step_transform
             )
 
+        Another example where the State Attributes ``trainer.state.alpha`` and ``trainer.state.beta``
+        are also logged along with the NLL and Accuracy after each iteration:
+
+        .. code-block:: python
+
+            clearml_logger.attach(
```
Comment (vfdev-5) [inline ignite/contrib/handlers/wandb_logger.py:234]:
```suggestion
            wandb_logger.attach(
```

File: ignite/contrib/handlers/wandb_logger.py [hunk 2]
```diff
@@ -226,6 +226,22 @@ def global_step_transform(*args, **kwargs):
...
+
+            wandb_logger.attach(
+                trainer,
+                log_handler=OutputHandler(
+                    tag="training",
+                    metric_names=["nll", "accuracy"],
+                    state_attributes=["alpha", "beta"],
+                ),
+                event_name=Events.ITERATION_COMPLETED
+            )
```
Comment (vfdev-5) [inline ignite/contrib/handlers/wandb_logger.py:242]: Actually, I think an example using `wandb_logger.attach_output_handler` would be shorter than `wandb_logger.attach`. Take a look at lines 221-226

Comment (Ishan-Kumar2) [reply ignite/contrib/handlers/wandb_logger.py:242]:
```suggestion
            wandb_logger.attach_output_handler(
                trainer,
                event_name=Events.ITERATION_COMPLETED,
                tag="training",
                metrics=["nll", "accuracy"],
                state_attributes=["alpha", "beta"],
            )
```
Do mean like this? It does reduce the line of initialising the Output Handler

Comment (vfdev-5) [reply ignite/contrib/handlers/wandb_logger.py:242]: Yes, that's correct

**Review by vfdev-5 — APPROVED** [review]
Thanks @Ishan-Kumar2

**Pull request merged**